www/~/%username/emails/verify.html.spt

```diff
@@ -4,14 +4,17 @@ from datetime import timedelta
 
 from aspen import Response
 from aspen.utils import utcnow
-from gratipay.utils import emails, get_participant
+from gratipay.utils import emails, get_participant, b64decode_s
 
 [-----------------------------------------------------------------------------]
 
 participant = get_participant(state, restrict=False)
 banner = '~' + participant.username
 if participant == user.participant:
-    email = request.qs.get('email', '')
+    if 'email' in request.qs:
+        email = request.qs['email']
+    else:
+        email = b64decode_s(request.qs.get('email64', ''), default='')
```

Comment:
> through [this](https://github.com/gratipay/gratipay.com/blob/master/www/~/%25username/emails/verify.html.spt#L76)

(Quick point of order: in GitHub file links, swap `master` for the [current version number](https://gratipay.com/version.txt) to get a nice permalink, to avoid drift.)